xbmc/cores/DllLoader/Win32DllLoader.cpp

```diff
@@ -36,75 +37,75 @@ struct Export
+  { "fseek",                      -1UL, (void*)dll_fseek          },
+  { "ftell",                      -1UL, (void*)dll_ftell          },
+  { "getc",                       -1UL, (void*)dll_getc           },
+  { "fgetc",                      -1UL, (void*)dll_getc           },
+  { "rewind",                     -1UL, (void*)dll_rewind         },
+  { "vfprintf",                   -1UL, (void*)dll_vfprintf       },
+  { "fgetpos",                    -1UL, (void*)dll_fgetpos        },
+  { "fsetpos",                    -1UL, (void*)dll_fsetpos        },
+  { "_stati64",                   -1UL, (void*)dll_stati64        },
+  { "_stat64",                    -1UL, (void*)dll_stat64         },
+  { "_stat64i32",                 -1UL, (void*)dll_stat64i32      },
+  { "_fstati64",                  -1UL, (void*)dll_fstati64       },
+  { "_fstat64",                   -1UL, (void*)dll_fstat64        },
+  { "_fstat64i32",                -1UL, (void*)dll_fstat64i32     },
+  { "_telli64",                   -1UL, (void*)dll_telli64        },
+  { "_tell",                      -1UL, (void*)dll_tell           },
+  { "_fileno",                    -1UL, (void*)dll_fileno         },
+  { "ferror",                     -1UL, (void*)dll_ferror         },
+  { "freopen",                    -1UL, (void*)dll_freopen        },
+  { "fscanf",                     -1UL, (void*)dll_fscanf         },
+  { "ungetc",                     -1UL, (void*)dll_ungetc         },
+  { "_fdopen",                    -1UL, (void*)dll_fdopen         },
+  { "clearerr",                   -1UL, (void*)dll_clearerr       },
   // for debugging
-  { "printf",                     -1, (void*)dllprintf          },
-  { "vprintf",                    -1, (void*)dllvprintf         },
-  { "perror",                     -1, (void*)dllperror          },
-  { "puts",                       -1, (void*)dllputs            },
+  { "printf",                     -1UL, (void*)dllprintf          },
+  { "vprintf",                    -1UL, (void*)dllvprintf         },
+  { "perror",                     -1UL, (void*)dllperror          },
+  { "puts",                       -1UL, (void*)dllputs            },
   // workarounds for non-win32 signals
-  { "signal",                     -1, (void*)dll_signal         },
+  { "signal",                     -1UL, (void*)dll_signal         },
 
   // libdvdnav + python need this (due to us using dll_putenv() to put stuff only?)
-  { "getenv",                     -1, (void*)dll_getenv         },
-  { "_environ",                   -1, (void*)&dll__environ      },
-  { "_open_osfhandle",            -1, (void*)dll_open_osfhandle },
+  { "getenv",                     -1UL, (void*)dll_getenv         },
+  { "_environ",                   -1UL, (void*)&dll__environ      },
+  { "_open_osfhandle",            -1UL, (void*)dll_open_osfhandle },
 
-  { NULL,                         -1, NULL                      }
+  { NULL,                         -1UL, NULL                      }
```

Comment:
`std::string_view` cannot be initialized to NULL (runtime crash).

I'm not sure if end array with NULL elements is still need, probably can be removed this end line. This was used in while loop:

`while (exp->name)`

but now is used for loop:

`for (const auto& exp : win32_exports)`

Reply:
Yep, I missed that. Thanks!